pallets/file-system/Cargo.toml

```diff
@@ -31,6 +32,7 @@ sp-trie = { workspace = true }
 
 # Local
 pallet-file-system-runtime-api = { workspace = true }
+pallet-proofs-dealer = { workspace = true }
```

Comment:
This shouldn't be here. This tightly couples the File System pallet to the Proofs Dealer pallet, instead of being loosely coupled (by an interface trait).